nix/wire-server.nix

```diff
@@ -459,9 +459,23 @@ let
       export LOCALE_ARCHIVE=${pkgs.glibcLocales}/lib/locale/locale-archive
     '';
   };
+  allLocalPackages = pkgs.symlinkJoin {
+    name = "all-local-packages";
+    paths = map (e: (hPkgs localModsEnableAll).${e}) wireServerPackages;
+  };
+
+  allImages = pkgs.linkFarm "all-images" (images localModsEnableAll) ;
+
+  # BOM is an acronym for bill of materials
+  allLocalPackagesBom = lib.buildBom allLocalPackages {
+    includeBuildtimeDependencies = true;
+  };
+  allImagesBom = lib.buildBom allImages {
+    includeBuildtimeDependencies = true;
+  };
```

Comment:
Does this work now?

Reply:
I'm not entirely sure but I *think* there are some things missing.